test/initialization-test/run_test.sh

```diff
@@ -25,6 +25,7 @@ usage()
     echo "usage: run_test.sh
     --results-gcs-dir GCS directory for the test results. Usually gs://<project-id>/<commit-sha>/initialization_test
     [--namespace      k8s namespace where ml-pipelines is deployed. The tests run against the instance in this namespace]
+    [--test-v2-api run test using v2 API]
```

Comment:
nit: make the space consistent with other options.

Reply:
Good catch. Fixed.